openxr/src/lib.rs

```diff
@@ -59,6 +59,12 @@ pub const USER_HEAD: &str = "/user/head";
 pub const USER_GAMEPAD: &str = "/user/gamepad";
 pub const USER_TREADMILL: &str = "/user/treadmill";
 
+// Helper traits
+pub trait AsHandle {
+    type Handle: sys::Handle;
+    fn handle(&self) -> Self::Handle;
```

Comment:
Slightly more idiomatic and consistent:
```suggestion
    fn as_handle(&self) -> Self::Handle;
```